src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -281,6 +281,13 @@
       <arg type="s" name="transaction_address" direction="out"/>
     </method>
 
+    <!-- Set options in yum .repo files -->
+    <method name="ModifyYumRepo">
+      <arg type="s" name="repo_id" direction="in"/>
+      <arg type="a{ss}" name="repo_options" direction="in"/>
```

Comment:
Bikeshed: maybe just `options`?

Reply:
Hm, I called it `repo_options` to avoid confusing it with `options` such as "dry-run" that other DBus API has. Not sure it's a huge problem :) Happy to change it if you think it's best.

Reply:
`settings` or `parameters` could work too? Just writing the polkit messages and "Authentication is required to modify repository settings" is what I have right now and "settings" seems to work well in that sentence.

Reply:
Sure, `settings` sounds good to me!